ext/afform/core/templates/CRM/Afform/Page/AfformBase.tpl

```diff
@@ -2,7 +2,10 @@
   <form id="bootstrap-theme" ng-controller="AfformStandalonePageCtrl">
     {literal}
       <h1 style="display: none" crm-page-title ng-if="afformTitle">{{ afformTitle }}</h1>
+      <div ng-if="!afformDisplayForm">
+        {{ afformConfirmationMessage }}
+      </div>
     {/literal}
-    <{$directive}></{$directive}>
-  </form>
+    <{$directive} {literal} ng-if="afformDisplayForm" {/literal}></{$directive}>
```

Comment:
@kurund I suspect the unit test is too brittle and is expecting exact markup which has now been changed by this PR.

That being said... I'm not sure changing this markup is actually the best approach. It looks like you've written this functionality into the afform standalone template, but that template is only used for full-page Afforms.

The idea behind Afforms is that they can be embedded anywhere: blocks, tabs, dashboards, popups, other afforms... and that's why this template is so thin: it's not meant to contain any logic or functionality: the Afform should be self-contained.

Placing the logic in this template means that an admin can configure confirmation messages for their forms, but then they won't actually work anywhere except full-screen pages.

Reply:
@colemanw 

Thank for the feedback. I was not under the impression that AfformBase.tpl was used everywhere.

Let me check.

Reply:
I have pushed the updated code.